src/sentry/lang/native/processing.py

```diff
@@ -539,19 +539,11 @@ def emit_apple_symbol_stats(apple_symbol_stats, data):
         # This is done to temporally collect information about the events for which symx is not working correctly.
         if in_random_rollout("symbolicate.symx-logging-rate") and os_name and os_version:
             os_description = os_name + str(os_version)
-            logger.info(
-                "Failed to find symbols using symx",
-                extra={
-                    "id": data.get("event_id"),
-                    "modules": old,
-                    "os_description": os_description,
-                    "os_filter": options.get("symbolicate.symx-os-description-list"),
-                },
-            )
             if os_description in options.get("symbolicate.symx-os-description-list"):
                 with sentry_sdk.isolation_scope() as scope:
                     scope.set_context(
-                        "Event Info", {"id": data.get("event_id"), "modules": str(old)}
+                        "Event Info",
+                        {"id": data.get("event_id"), "modules": old, "os": os_description},
```

Comment:
Can context values be arbitrary data?

Reply:
Looking at the type signature that is what I would be expecting: 
```py   
 def set_context(
        self,
        key,  # type: str
        value,  # type: Dict[str, Any]
    ):
        # type: (...) -> None
        """
        Binds a context at a certain key to a specific value.
        """
        self._contexts[key] = value
```

Reply:
Right, https://docs.sentry.io/platforms/python/enriching-events/context/#structured-context also says so.